src/test/java/org/opentripplanner/smoketest/SeptaSmokeTest.java

```diff
@@ -8,8 +8,6 @@
 
 /**
  * This smoke test expects an OTP installation running at localhost:8080
- * <p>
- * It uses the REST API to check that a route from Philadelphia's airport to its North can be found.
```

Comment:
Why was this removed?

Reply:
It was outdated and didn't reflect anymore the content of the test. I didn't want to repeat the list of tests below either. Did you find it useful?

Reply:
Makes sense. Can be revamped!